webextension/background.js

```diff
@@ -542,3 +544,10 @@ browser.tabs.onUpdated.addListener(function(id, changeinfo, tab) {
 	overlay_ready[id] = false;
     }
 });
+
+browser.tabs.onRemoved.addListener(function(removedTabId) {
+    // delete from overlay readiness, tab CSS status and window_height
```

Comment:
remove tabId when tab is closed

Reply:
I guess this is more clean, though the memory savings will probably be negligible. It is my understanding that tab IDs are unique to a session, so stale values should not be a problem. But I don't see any harm in this cleanup, so just fix the indentation on the first line and this is OK.

Reply:
I've pushed an update.
I've remove the console.log and the else branch.
I've replace some `let` by `const` when needed.
I've formated the code with vscode but it may not be correct for you, let me know I'll do a rollback.

btw you should provide a linter rules in your code this way it will be easier to follow tour coding rules.
I use eslinter with vscode, I can send you my rules file as example if you are interested.